### ISSUE TITLE
tests import error on python 3.5.1, stock install/venv

### ISSUE THREAD

**Issue by ctb** [jdidion/xphyle#9]
% python setup.py test

```
...
  File "/Users/t/dev/xphyle/tests/test_formats.py", line 4, in <module>
    from xphyle.formats import *
  File "/Users/t/dev/xphyle/xphyle/__init__.py", line 13, in <module>
    from xphyle.formats import FORMATS, THREADS
  File "/Users/t/dev/xphyle/xphyle/formats.py", line 12, in <module>
    from xphyle.paths import (
  File "/Users/t/dev/xphyle/xphyle/paths.py", line 13, in <module>
    from xphyle.types import (
  File "/Users/t/dev/xphyle/xphyle/types.py", line 346, in <module>
    AnyChar = Union[bytes, Text]
NameError: name 'Text' is not defined 
```

**Comment by ctb** [comment]
This turns out to cause problems with a basic import, `from xphyle import *`. Suggestions on how to proceed are welcome :). Otherwise I'll spend some time googling about.

**Comment by jdidion** [comment]
Working on it now.

**Comment by jdidion** [comment]
Oof - so this change was introduced in 3.5.2. Unfortunately there's no mention of that in the docs, so I assumed that it had been around since 3.5. I will fix this by moving to the new 'typing' module (https://pypi.org/project/typing/) and adding it as a dependency for python < 3.6.

**Comment by jdidion** [comment]
Fixed in 3.0.2. The fix for this isn't pretty, but it works: I have to force the use of backports.typing for versions prior to 3.6. 

On a side note: It's looking like 3.6 is going to be regarded as the first feature-stable 3.x version, so I'm hoping most people on 3.x will upgrade at some point in the near future. I'll probably drop support for pre-3.6 at the next major release (4.0) - which probably won't be for a while, as I consider the xphyle API to be stable as of 3.0.

**Comment by ctb** [comment]
fixed!
